test/ibmq/test_ibmq_job.py

```diff
@@ -377,20 +377,34 @@ def test_retrieve_jobs_between_datetimes_not_overriden(self, provider):
     def test_retrieve_jobs_filter_counts(self, provider):
         """Test retrieving jobs filtered by counts."""
         # TODO: consider generalizing backend name
-        # TODO: this tests depends on the previous executions of the user
         backend = provider.get_backend('ibmq_qasm_simulator')
 
-        my_filter = {'backend.name': 'ibmq_qasm_simulator',
-                     'shots': 1024,
-                     'qasms.result.data.counts.00': {'lt': 500}}
+        # Submit jobs with desired attributes.
+        jobs = []
+        qc = QuantumCircuit(3, 3)
+        qc.h(0)
+        qc.measure([0, 1, 2], [0, 1, 2])
+        qobj = assemble(transpile(qc, backend=backend), backend=backend)
+        for _ in range(2):
+            job = backend.run(qobj)
+            job.result()
+            jobs.append(job)
+
+        my_filter = {'backend.name': backend.name(),
+                     'summaryData.summary.qobj_config.n_qubits': 3,
+                     'status': 'COMPLETED'}
 
         job_list = provider.backends.jobs(backend_name=backend.name(),
-                                          limit=5, skip=0, db_filter=my_filter)
+                                          limit=2, skip=0, db_filter=my_filter)
+        self.assertEqual({o_job.job_id() for o_job in jobs},
+                         {r_job.job_id() for r_job in jobs})
```

Comment:
Is `jobs` on line `400` supposed to be `job_list`?

Reply:
Yes